### PR TITLE
Protocol: Add capture_timestamp to fdpay header

### PR DESCRIPTION
**Pull request by wmanley**
The capture_timestamp is the time at which the video-frame was captured
(e.g. from v4l) as measured against the Linux CLOCK_MONOTONIC clock.  This
is useful for synchronisation between multiple sources.  the monotonic
clock was chosen because that is the format most v4l and alsa devices
provide their data in.

This is a backward incompatible change to the wire-protocol so I have
bumped the DBus interface version number.  Old versions and new versions
of pulsevideo will not interoperate, but at least they will fail early and
loudly.

### CODE REVIEW

**Comment by wmanley** [comment]
@drothlis: Please review

File: tests/integration_test.py [hunk 1]
```diff
@@ -296,3 +297,26 @@ def test_that_invalid_sized_buffers_are_dropped(tmpdir):
...
+                ['gst-launch-1.0', '-q', 'pulsevideosrc',
+                 'bus-name=com.stbtester.VideoSource.test', '!', 'identity',
+                 'error-after=11', '!', 'checksumsink'])
+        except subprocess.CalledProcessError as e:
+            output = e.output
+    buffers = [x.split() for x in output.strip().split('\n')]
+    timestamps = [parse_gst_timestamp(x[0]) for x in buffers]
+    assert len(timestamps) == 10
+    for n in range(1, 9):
+        assert abs(timestamps[n] + 0.1 - timestamps[n + 1]) < 1e-9
```
Comment (drothlis) [inline tests/integration_test.py]: This testcase would be clearer if the videotestsrc framerate (which defaults to 10fps) is stated explicitly in the source_pipeline.

**Comment by drothlis** [comment]
Looks good to me, though my GStreamer & GObject knowledge isn't sufficient to give this a thorough review.

File: tests/integration_test.py (hunk 1 above)
Comment (wmanley) [inline tests/integration_test.py]: I've put in a comment to that effect

**Pull request merged**